src/tools/xtp_run.cc

```diff
@@ -58,7 +58,8 @@ bool XtpRun::EvaluateOptions() {
   std::string helpdir = "xtp/xml";
   if (OptionsMap().count("list")) {
     std::cout << "Available XTP calculators:\n";
-    for (const auto& name : xtp::Calculators().getKeys()) {
+    xtp::Calculatorfactory inst;
```

Comment:
can we not just create one instance in the function scope? Instead of using three different factories?

Reply:
sure